### PR TITLE
libbpf-tools: Support verbose mode and set custom libbpf print callback

### PR DESCRIPTION
**Pull request by chenhengqi**
Add verbose option and set custom libbpf print callback in all tools.
This would allow user to print libbpf debug log and aid the debugging
process.

### CODE REVIEW

**Comment by chenhengqi** [comment]
@davemarchevsky Please take a look.  Thanks.

File: libbpf-tools/ksnoop.c [hunk 1]
```diff
@@ -251,7 +252,7 @@ static int get_func_btf(struct btf *btf, struct func *func)
 	return 0;
 }
 
-int predicate_to_value(char *predicate, struct value *val)
```
Comment (davemarchevsky) [inline libbpf-tools/ksnoop.c:254]: I'm fine with this change, but can you confirm that it's independent of the others in this file? If it's not, then I must be confused

**Review by davemarchevsky — APPROVED** [review]
LGTM aside from one question

File: libbpf-tools/ksnoop.c (hunk 1 above)
Comment (chenhengqi) [reply libbpf-tools/ksnoop.c:254]: Yes, it's independent.

**Pull request merged**